### PR TITLE
Test/footer

### PR DESCRIPTION
**Pull request by sjpaige**
This is the first test set for the footer element. 

I have the test configuration working on my mac with jest/enzyme after almost an entire day of de-bugging different configurations, and I am afraid of how delicate the setup is. If the merging the newer staging code to the enzyme-master code isn't working out then I can begin writing tests using [react testing library ](https://testing-library.com/docs/react-testing-library/intro/) for rendering instead of enzyme.

### CODE REVIEW

**Comment by stuttskl** [comment]
Nice job so far! Hopefully adding enzyme to the project wasn't too terribly burdensome. 

Let me know what you think of this--how about if we put a test directory in each folder? For example, if we were to have a "testing" folder in the "Components" directory and have our test files there.

File: src/Tests/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import Footer from '../Components/Footer'
+
+describe("Footer", () => {
+
+    const footer = shallow(<Footer />);
```
Comment (stuttskl) [inline src/Tests/Footer.test.js:7]: I _think_ this might be valid syntax... But can we put this line in a `beforeEach`?

Comment (sjpaige) [reply src/Tests/Footer.test.js:7]: Yes we can. Thanks for catching that.

**Comment by sjpaige** [comment]
> 
> 
> Nice job so far! Hopefully adding enzyme to the project wasn't too terribly burdensome.
> 
> Let me know what you think of this--how about if we put a test directory in each folder? For example, if we were to have a "testing" folder in the "Components" directory and have our test files there.


No worries! Spending the time figuring Enzyme out compatibility was a great learning experience not a burden. 

I just wanted to make sure that rolling back to React 16 isn't going to cause issues with the other code that is already on staging, and the other libraries we depend on. I was mentioning the React Testing Library so that you knew I was willing to fallback on it if the Enzyme changes are causing conflicts or bugs that are more trouble to fix than me just changing over to RTL and keeping React 17 + current dependencies.

We can definitely move the tests to be with their components/pages, which is closer to the current best practices, I had just put it there since I was having trouble with the Jest code detection in vscode and it needed that json helper file to run before the test.

**Review by stuttskl — APPROVED** [review]
Looks great! Approved pending branch conflicts.

**Pull request closed without merging**